components/types/types.ts

```diff
@@ -1258,3 +1258,13 @@ export type ContextScopedStorageLocation = "profile" | "hit" | "session"
 export type CPDStore = Record<string, Record<string, string | number | boolean>>
 
 export type ContractProgressionData = Record<string, string | number | boolean>
+
+/**
+ * LiveSplit-related types
+ */
+export interface LiveSplitTimeCalcEntry {
```

Comment:
Wondering if types.ts is the place for this. Not sure what the policy is in this regard, but this seems specific to LiveSplit and not to types from the game. Shouldn't it go/remain in the livesplit sub-folder then?

Reply:
I decided to move it to a new `livesplit.ts` types file but I think it's a bit too much for only one interface, as we don't use more of those on this specific module.

Perhaps, maybe the interface could live in the module code itself?
I'd love to get more feedback on this as I don't really know how we should go with this.

Thoughts @RDIL ?

Reply:
I don't really have a specific preference, they can go wherever tbh

Reply:
I'll leave it like this then. We'll probably discuss this later if we have a need to re-organize typings for whatever reason.